wsi/drm/drmdisplay.cpp

```diff
@@ -37,6 +37,7 @@
 #define CTA_EXTENSION_TAG 0x02
 #define CTA_EXTENDED_TAG_CODE 0x07
 #define CTA_COLORIMETRY_CODE 0x05
+int FirstFrame=1;
```

Comment:
what is this used for? just see it set to 0 and print the log.

Reply:
only the first successful commit will be logged, used to check the 1st frame commit time.
This can be remove later, once the seamless work done.